### DISCUSSION TITLE
[A-L] (2023/24) Foglio 12 - Esercizio 9

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#274] [Esercizi - Viaggi/Piazza]
**Esercizio 5.2 del libro di testo (Geometria Analitica: Marco Abate & Chiara De Fabritiis)**

**Comment by daromacarlo** [comment]
[es 5.2 .pdf](https://github.com/sapienzastudentsnetwork/algebra2324/files/13548598/es.5.2.pdf)

**Comment by MarcoLinardi** [comment]
- Per quanto riguarda S2 possiamo notare come all'interno di una matrice non può esserci la stessa variabile del vettore, quindi S1 non è lineare
- Per quanto riguarda S2 possiamo notare come non possa verificarsi una somma per una costante come risultato del prodotto tra una matrice e un vettore, quindi S2 non è lineare
